src/adaptors/curve/index.js

```diff
@@ -1,269 +1,87 @@
-        tvl += poolData.balances[1] / decimals[1];
-      } else if (pool === 'xautusd') {
-        tvl = (poolData.balances[0] / decimals[0]) * poolData.price_scale;
-        tvl += poolData.balances[1] / decimals[1];
-      } else {
-        tvl = 0;
-        poolData.balances.forEach((el, i) => {
-          tvl += (el / decimals[i]) * poolData.crypto_prices[i];
-        });
+    if (response?.success) {
+      for (const pool of response.data.poolData) {
+        poolInfo[pool.address] = {
+          symbol: pool.coins.map((coin) => coin.symbol).join('-'),
+          tvlUsd: pool.usdTotal,
+        };
       }
-    } else {
-      // for all else
-      tvl = poolData.supply * price;
     }
-
-    // scale by virtual price
-    tvl *= poolData.virtual_price;
-
-    let apyBase = apy * 100;
-    // some of the values are negative, set to 0
-    apyBase = apyBase < 0 ? 0 : apyBase;
-
-    // 4) get crv+reward apr data
-    let dataCrvApy = await utils.getData(
-      'https://www.convexfinance.com/api/curve-apys'
-    );
-    let dataRewardApy = await utils.getData('https://api.curve.fi/api/getApys');
-    dataCrvApy = dataCrvApy.apys;
-    dataRewardApy = dataRewardApy.data;
-
-    // unfort these two endpoints have different keys for some pools
-    let searchKey = pool;
-    if (pool === 'y') {
-      searchKey = 'iearn';
-    } else if (pool === 'susd') {
-      searchKey = 'susdv2';
-    } else if (pool === 'ren2') {
-      searchKey = 'ren';
-    }
-
-    const apyCrv = dataCrvApy[searchKey]?.crvApy;
-
-    // need to sum up the potential rewards apys
-    const apyRewardsArray = dataRewardApy[searchKey]?.additionalRewards;
-    let apyRewardsSum = 0;
-    if (apyRewardsArray?.length > 0) {
-      apyRewardsSum = apyRewardsArray
-        .map((el) => el.apy)
-        .reduce((accumulator, curr) => accumulator + curr);
-    }
-    const apySum =
-      apyRewardsSum === undefined
-        ? apyBase + apyCrv
-        : apyBase + apyCrv + apyRewardsSum;
-
-    // 6) append output to object
-    poolStats.push({
-      pool,
-      virtual_price: poolData.virtual_price,
-      totalSupply: poolData.supply,
-      tvl,
-      apyBase,
-      apyCrv,
-      apyRewardsArray,
-      apyRewardsSum,
-      apy: apySum,
-    });
   }
-
-  // note(temporary only, investiage 4pool and 2pool apy)
-  return poolStats.filter((el) => el.pool !== '4pool' && el.pool !== '2pool');
+  return poolInfo;
 };
 
-const getDataEVM = async (chainString) => {
-  let tvl = await utils.getData(
-    `https://api.curve.fi/api/getTVL${
-      chainString.charAt(0).toUpperCase() + chainString.slice(1)
-    }`
-  );
-  tvl = tvl.data.pools;
-
-  const poolNames = Object.keys(tvl);
-  let dataTvl = [];
-  for (const p of poolNames) {
-    const obj = tvl[p];
-    obj['pool'] = p;
-    dataTvl.push(obj);
+const getCurvePoolsAPY = async (blockchainId) => {
+  const uri = `/getSubgraphData/${blockchainId}`;
```

Comment:
This subgraph has only baseApy in percents so there is no need to multiply by 100
<img width="499" alt="Screenshot 2022-07-11 at 18 23 33" src="https://user-images.githubusercontent.com/31341205/178299430-16654317-84b2-46f4-b664-8c0f504c0cbe.png">
<img width="503" alt="Screenshot 2022-07-11 at 18 23 55" src="https://user-images.githubusercontent.com/31341205/178299519-af8b7135-20d8-419e-a8f9-2846d1ae1d92.png">